mitiq/benchmarks/ghz_circuits.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2022 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Functions for creating GHZ circuits for benchmarking purposes."""
+
+# imports
+from typing import Optional
+from mitiq import QPROGRAM
+import cirq
+
+from mitiq.interface import convert_from_mitiq
+
+
+def generate_ghz_circuit(
+    n_qubits: int, return_type: Optional[str] = None,
+) -> QPROGRAM:
+    """ Returns a GHZ circuit ie a circuit that prepares an n_qubits GHZ state.
```

Comment:
Nit: Remove space, put `n_qubits` in two backticks (as is done with cirq.Circuit) here and below in the docstring.